README.md

```diff
@@ -4,15 +4,21 @@ For the most up-to-date information on Anchore Engine, Anchore CLI, and other An
 
 The Anchore Engine is an open-source project that provides a centralized service for inspection, analysis, and certification of container images. The Anchore Engine is provided as a Docker container image that can be run standalone or within an orchestration platform such as Kubernetes, Docker Swarm, Rancher, Amazon ECS, and other container orchestration platforms.
 
+In addition, we also have several modular container tools that can be run standalone or integrated into automated workflows such as CI/CD pipelines.
```

Comment:
We may have several other tools, but this is listing syft and grype. This could generalize instead of quantifying, for example: 

> In addition, we have container tooling [...]

But I would go further and not emphasize "container" because these also work for filesystems - or this could mention both, and omit from repeating it in the descriptions below

Reply:
What would be an example of an _"integrated automated workflow"_ that isn't something related to a CI/CD service? I think this makes it wordy

Reply:
@alfredodeza These suggestions come from @kimweins, so your questions may be better suited for her. Check out the linked Slack thread in the PR description if you haven't already.

My two cents:

- I'm up for tweaking this.
- For writing, I prefer the word "tools" over "tooling". While the latter is not wrong, it's a bit inflated and less clear than "tools" which works just as well here.
- Re: kinds of automated workflows, there are plenty of automated workflows that aren't CI/CD. Having said that, I agree that this might be a bit wordy, and we could just refer to CI/CD workflows directly here.